rocAL/rocAL/source/meta_node_flip.cpp

```diff
@@ -49,31 +49,38 @@ void FlipMetaNode::update_parameters(MetaDataBatch* input_meta_data)
         memcpy(coords_buf, input_meta_data->get_bb_cords_batch()[i].data(), input_meta_data->get_bb_cords_batch()[i].size() * sizeof(BoundingBoxCord));
         BoundingBoxCords bb_coords;
         BoundingBoxLabels bb_labels;
+        
         for(uint j = 0, m = 0; j < bb_count; j++)
         {
             BoundingBoxCord box;
-            box.x = coords_buf[m++];
-            box.y = coords_buf[m++];
-            box.w = coords_buf[m++];
-            box.h = coords_buf[m++];
+            box.l = coords_buf[m++];
+            box.t = coords_buf[m++];
+            box.r = coords_buf[m++];
+            box.b = coords_buf[m++];
+            
             if(_flip_axis_val[i] == 0)
             {
-                float centre_x = _src_width_val[i] / 2;
-                box.x += ((centre_x - box.x) * 2) - box.w;
+                float l = 1 - box.r;
```

Comment:
remove extra variables and do inline computation

Reply:
I would required float l here , removed the float r

If if do it the below way
box.l = 1- box.r
box.r = 1-box.l
For computing box.r , i would be using the updated r co-ordinates whereas i require the values before they are update for mirror